inst/extdata/3.14/grieneisen-ts

```diff
@@ -0,0 +1 @@
+
```

Comment:
This is a file, but it needs to be a folder with the metadata inside. See the other folders for an example

Reply:
Should i cancel this pull request and make a new one then? Because it doesn't allow me to change from "inst/extdata/3.14/grieneisen-ts" to "inst/extdata/3.14/grieneisen-ts/make-TreeSE-object.R" and says that "A file exists where you’re trying to create a subdirectory. Choose a new path and try again."

Reply:
You have to delete the file and then create the directory. Also please not that in `inst/extdata/` only csv files should be put. These csv files should be the product of the `make-some-metdata.R` files in `inst/scripts/`.

The script you are working on is something else and is optional: A script to assemble the data in the first place. This is nice to have, but not a requirement for the ExperimentHub package. The other files mentioned above are not optional.

Reply:
I created a folder to keep r script and csv file separate from the already existing files. Would it be better if i directly insert them into "inst/scripts" for R script and "inst/extdata" for csv file?